src/sentry/lang/native/sources.py

```diff
@@ -355,10 +400,12 @@ def parse_sources(config):
     except Exception as e:
         raise InvalidSourcesError("Sources are not valid serialised JSON") from e
 
-    validate_sources(sources)
-
     # TODO(@anonrig): Remove this when AppStore connect related datas are removed.
```

Comment:
please remove this TODO as well, as we would want to keep the filtering indefinitely.